src/commands/ExecCommand.js

```diff
@@ -18,17 +18,19 @@ export const builder = {
     group: "Command Options:",
     describe: "Bail on exec execution when the command fails within a package",
     type: "boolean",
-    default: true,
+    default: undefined,
```

Comment:
ugh, this noise really bothers me, should be fixed upstream.

Glad you figured it out, all the same. :)